cfgen/cfgen

```diff
@@ -1513,6 +1563,93 @@ class ConfProfile(ToDhall):
         return prof_id
 
 
+class ConfFdmiFltGrp(ToDhall):
+    _objt = ObjT.fdmi_flt_grp
+    _downlinks: Set[Downlink] = set()
+
+    def __init__(self, rec_type: int, fdmi_filters: List[Oid]):
+        assert all(x.type is ObjT.fdmi_filter for x in fdmi_filters)
+        self.rec_type = rec_type
+        self.fdmi_filters = fdmi_filters
+
+    def __repr__(self):
+        args = ', '.join([f'rec_type={self.rec_type}',
+                          f'fdmi_filters={self.fdmi_filters!r}'])
+        return f'{self.__class__.__name__}({args})'
+
+    def to_dhall(self, oid: Oid) -> str:
+        assert oid.type is ObjT.fdmi_flt_grp
+        args = ', '.join([f'id = {oid_to_dhall(oid)}',
+                          f'rec_type = {self.rec_type}',
+                          f'filters = {oids_to_dhall(self.fdmi_filters)}'])
+        return '{ %s }' % args
+
+    @classmethod
+    def build(cls, m0conf: Dict[Oid, Any], parent: Oid,
+              fdmi_filter_desc: Dict[str, Any],
+              other_clients: Dict[str, List[Tuple[Oid, Oid]]]) -> Oid:
+        assert parent.type is ObjT.root
+
+        fdmi_flt_grp_id = new_oid(ObjT.fdmi_flt_grp)
+        fdmi_filter_oid = ConfFdmiFilter.build(
+            m0conf, parent, fdmi_flt_grp_id, fdmi_filter_desc, other_clients)
+        # XXX rec_type is M0_FDMI_REC_TYPE_FOL
```

Comment:
I don't quite understand this comment. Is it something that we need to check in the code somehow (i.e. missing assert)?
Or it just explains magical `rec_type=0x1000`?

Reply:
Added better explanation.